backend excercise/participation/models.py

```diff
@@ -0,0 +1,63 @@
+import uuid
+from django.db import models
+
+from quizzes.models import Question, Quiz, Subject
+from users.models import User
+
+# Create your models here.
+
+class UserQuiz(models.Model):
+    User_Quiz_Id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
```

Comment:
rename it to ID

Reply:
create a single class containing common fields like
Id, is_deleted, created_at, last_modified_at,